### PR TITLE
Create NOOP pptm.js script upon setup

### PR DESCRIPTION
**Pull request by brunops**
Create async tag manager script upon setup, works as a NOOP. The idea is that it can be used later on to power new paypal integrations *after* opt-in with no effort.

- Update `src/loader/util#loadScript` so it accepts custom attributes. Used for `id` and `async` when creating `pptm.js` script
- Serve empty `/tagmanager/pptm.js` file through Karma server so there are no 404s when running the tests
- Do not load pptm.js when inside paypal domain. This is necessary so it doesn't end up loaded twice, one in the third-party website, and another inside the iframe
- Add logger calls when creating pptm.js tag and make sure not to include it twice on the same page
- Log when trying to load and after loaded
- Also log if the there was an attempt to create the pptmjs tag twice

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -35,6 +35,12 @@ module.exports = function(config) {
             },
 
             {
+                pattern: 'test/lib/pptm.js',
```
Comment (bluepnume) [inline karma.conf.js]: Could you just include with a regular `export * from './pptm.js'` to match the other test files?

Comment (brunops) [inline karma.conf.js]: This is something else, the idea is to setup karma to serve this empty pptm.js file. So there's no 404 when the tests run, and the script is added to the DOM.

Comment (bluepnume) [inline karma.conf.js]: Oh my bad -- sounds good then

**Comment by bluepnume** [comment]
Can't see a single problem -- let me know if you're ready for me to merge, then we can figure out a time this week to do a release!

**Pull request merged**